samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Scene.razor

```diff
@@ -132,5 +133,6 @@
     private SceneView? _view;
     private bool _rendered;
 
+    [SuppressMessage("ReSharper", "NotAccessedPositionalProperty.Local")]
```

Comment:
I had to look this up.  Is there a reason to keep the testObject in this file vs put in the Shared/Pages/Tests.razor or some other file?